lib/driver/pg.js

```diff
@@ -290,8 +290,21 @@ var PgDriver = Base.extend({
         } else {
           sql = util.format('ALTER TABLE "%s" ALTER COLUMN "%s" DROP DEFAULT', tableName, columnName);
         }
+        return this.runSql(sql).nodeify(setType.bind(this));
```

Comment:
Suggestions moving forward? Should I commit my pull request or should we scrap this until you get around to it? Mind you, tests are still passing.

Any other suggestions are welcome too. I'm happy to help!